manifests/repo/debian.pp

```diff
@@ -6,7 +6,7 @@
 
     apt::source { 'collectd-ci':
       location => 'https://pkg.ci.collectd.org/deb/',
-      repos    => "collectd-${$::collectd::ci_package_repo}",
```

Comment:
Did this ever work at all?? (too many `$`s).

Reply:
I do not have a debian box to test on. We are a CentOS/RedHat shop.